src/client/auth/scram.rs

```diff
@@ -57,19 +57,32 @@ struct CacheEntry {
 }
 
 /// The versions of SCRAM supported by the driver (classified according to hash function used).
-#[derive(Hash, Eq, PartialEq, Clone)]
+#[derive(Debug, Hash, Eq, PartialEq, Clone)]
```

Comment:
While testing this, I found it useful to have Debug on a struct that contained this